src/huggingface_hub/inference/_async_client.py

```diff
@@ -0,0 +1,1269 @@
+# coding=utf-8
+# Copyright 2023-present, the HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# WARNING
+# This entire file has been generated automatically based on `src/huggingface_hub/inference/_client.py`.
+# To re-generate it, run `make style` or `python ./utils/generate_async_inference_client.py --update`.
+# WARNING
```

Comment:
I think I'd either add a pretty visible disclaimer that any code written in this file will be overwritten by `make style`, or I would move this file entirely to a `_generated` folder to ensure that it's not being modified by someone (with the correct links for imports as well).

Reply:
I like the idea of a `_generated/` folder! Far better than just a tiny disclaimer that no-one will read. Thanks for the suggestion :)